src/betterproto/__init__.py

```diff
@@ -634,7 +655,19 @@ def __bytes__(self) -> bytes:
         return output + self._unknown_fields
 
     # For compatibility with other libraries
-    SerializeToString = __bytes__
+    def SerializeToString(self: T) -> bytes:
```

Comment:
Is it necessary to creating a wrapping function to document an method alias?

What if you just stick the docstring above attribute declaration?

Reply:
I think it improves readability and the same thing could be said about `FromString` could it not? As it could be done with `FromString = classmethod(parse)`.

Reply:
Readability for the docs or the code? FromString is different because it's wrapping an instance method in a class method.

Reply:
I think it's just better to be more consistent, keep the code more readable and make it slightly clearer what is happening.